### PR TITLE
DynamicEnable Property

### PR DESCRIPTION
**Pull request by TheColaber**
Resolvish #1673 - `editor-left` is now the only addon that can auto-refresh.
Also Resolvish #1672

### CODE REVIEW

File: addons/editor-stage-left/addon.json [hunk 1]
```diff
@@ -13,13 +13,14 @@
...
   "dynamicDisable": true,
+  "dynamicEnable": true,
   "userstyles": [
     {
       "url": "stageleft.css",
       "matches": ["https://scratch.mit.edu/projects/*"]
     }
   ],
-  "tags": ["editor", "theme"],
+  "tags": ["editor"],
```
Comment (mxmou) [inline addons/editor-stage-left/addon.json:23]: This addon was always in the themes category, this would remove it.

Comment (WorldLanguages) [reply addons/editor-stage-left/addon.json:23]: @mxmou See changes for settings page, the category was hardcoded to "theme" for this addon, even tho it lacks the tag.

**Comment by TheColaber** [comment]
closing... because of new system I will make a new PR soon ;)

**Comment by TheColaber** [comment]
See https://github.com/ScratchAddons/ScratchAddons/pull/1961 instead.

**Pull request closed without merging**